helm_deploy/keyworker-api/values.yaml

```diff
@@ -4,11 +4,75 @@
 generic-prometheus-alerts:
   targetApplication: keyworker-api
 
+generic-service:
+  image:
+    repository: quay.io/hmpps/keyworker-api
+    tag: latest
+    port: 8080
+
+  env:
+    JAVA_OPTS: "-Xmx512m"
+    SERVER_PORT: "8080"
+    SPRING_PROFILES_ACTIVE: "batch,sqs"
+    SERVER_CONNECTION_TIMEOUT: "180000"
+    APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY)"
+    APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
```

Comment:
Do we need this?

Reply:
other ones have this so was going to leave it the same as ones that work?